ElementX/Sources/Screens/RoomScreen/View/Style/TimelineItemBubbledStylerView.swift

```diff
@@ -98,6 +97,12 @@ struct TimelineItemBubbledStylerView<Content: View>: View {
         // Figma overlaps reactions by 3
         VStack(alignment: alignment, spacing: -3) {
             messageBubble
+                .accessibilityRepresentation(representation: {
+                    VStack {
+                        Text(timelineItem.sender.displayName ?? timelineItem.sender.id)
+                        messageBubble
+                    }
+                })
```

Comment:
Should be able to use a trailing closure here.
```suggestion
                .accessibilityRepresentation {
                    VStack {
                        Text(timelineItem.sender.displayName ?? timelineItem.sender.id)
                        messageBubble
                    }
                }
```